### ISSUE TITLE
Add 'pretty' option

### ISSUE THREAD

**Issue by se-panfilov** [ajwhite/gulp-ng-config#26]
All constants now is a single line, but it's little bit hard to track changes in config if it little more deeper:

Json file: 

```json
 {
  "ACTIONS": {
    "list": {
      "click": {
        "description": "just a click",
        "params": {
          "some": null,
          "another": false
        }
      }
  ...
}
```

What we have now:

```javascript
.constant('ACTIONS', {"list":{"click":{"description": "just a click","params":{"some": null,"another": false}}}})
```

What i'm expext:

```javascript
.constant('ACTIONS', {
  "list": {
    "click": {
      "description": "just a click",
      "params": {
        "some": null,
        "another": false
      }
    },
```

**Comment by Light241** [comment]
Added PR for this issue: https://github.com/ajwhite/gulp-ng-config/pull/27

**Comment by ajwhite** [comment]
Closed in #27
